Dockerfile

```diff
@@ -81,7 +81,11 @@ RUN cd /usr/src/lxc \
 	&& ldconfig
 
 # Install Go
-ENV GO_VERSION 1.4.2
+ENV GO_VERSION 1.5
+RUN curl -sSL https://golang.org/dl/go1.4.2.src.tar.gz | tar -v -C /usr/local -xz \
+	&& cd /usr/local/ && mv go go1.4.2
+RUN cd /usr/local/go1.4.2/src/ && ./make.bash
+ENV GOROOT_BOOTSTRAP /usr/local/go1.4.2
```

Comment:
@tianon It's mostly to avoid using third party binaries so that nobody frowns upon how this is built. It's also consistent with how we were doing things before 1.5: building it all from source. That's pretty much it.